feature-dogfooding-guidelines.md

```diff
@@ -0,0 +1,50 @@
+# Feature Dogfooding Guidelines
```

Comment:
I know that "Dogfooding" was a contentious name for us, haha, because it's not what dogfooding really is. I think that's when employees at a company use the software they develop themselves as actual users.

I heard "Bug bash" from MPFML and thought that was a better name! Plus - alliteration!

Reply:
I can change this to bug bash!